client/components/CircleIcon/index.coffee

```diff
@@ -8,7 +10,7 @@ module.exports = component
       PropTypes.number
     ]).isRequired
     background: PropTypes.string
-
+  css: css
```

Comment:
@contra it seems to be after displayName: https://github.com/wearefractal/product-boilerplate/pull/10/files#diff-258fef048fec1baa7e4da486f557f9d0R6
Unless I am mistaken by what you want after